data/assets/educational/scale/big_ben.asset

```diff
@@ -1,32 +1,35 @@
 local earthAsset = asset.require('scene/solarsystem/planets/earth/earth')
 local sunAsset = asset.require('scene/solarsystem/sun/sun')
 
+
+
 local modelFolder = asset.syncedResource({
   Name = "Scale Model Big Ben",
   Type = "HttpSynchronization",
   Identifier = "scale_model_big_ben",
   Version = 1
 })
 
-local scaleModel = {
+local Location = { 51.50079, -0.124279 }
+
+local ScaleModel = {
   Identifier = "Scale_BigBen",
   Parent = earthAsset.Earth.Identifier,
-  --Note: Lat/Lon/Scale values comes from alignment with Esri World Imagery 2D layer
+  -- Note: Lat/Lon/Scale values comes from alignment with Esri World Imagery 2D layer
   Transform = {
     Translation = {
       Type = "GlobeTranslation",
       Globe = earthAsset.Earth.Identifier,
-      Latitude = 51.50079,
-      Longitude = -0.124279,
+      Latitude = Location[1],
+      Longitude = Location[2],
       Altitude = 0.0,
       UseHeightmap = true
     },
     Rotation = {
       Type = "GlobeRotation",
       Globe = earthAsset.Earth.Identifier,
-      Latitude = 51.50079,
-      Longitude = -0.124279,
-      UseHeightmap = false
```

Comment:
Oops, no

Reply:
Assuming you mean the `UseHeightMap`?  Yes, since `false` is the default value and there is no reason to enforce the heightmap to be false in case we want to enable it by default later